src/cron/bundle.rs

```diff
@@ -100,23 +100,14 @@ where
     let mut current_block: Option<i64> = None;
     if let Err(err) = block_ok {
         return Err(err);
-    } else {
-        current_block = block_ok.unwrap();
     }
     if current_block.is_none() {
         return Ok(());
     }
 
     let path = arweave.get_tx_data(&bundle.id).await;
-    let mut file_path: Option<String> = None;
-    if path.is_ok() {
-        file_path = Some(path.unwrap());
-    } else {
-        error!("File path error {:?}", path);
-        return Ok(());
-    }
+    let path_str = path.unwrap();
```

Comment:
Maybe we should not return `Ok(())` in case of error, but instead an `Err` that allows server to return server error to the client who made the request.